src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Stylesheets with a .scss extension won't work directly in the browser. You should link to the compiled .css file instead.